test/unit/app/controllers/incoming-transactions-test.js

```diff
@@ -2,38 +2,51 @@ import assert from 'assert'
 import sinon from 'sinon'
 import proxyquire from 'proxyquire'
 
-import { ROPSTEN, RINKEBY, KOVAN, GOERLI, MAINNET } from '../../../../app/scripts/controllers/network/enums'
+import {
+  GOERLI,
+  KOVAN,
+  MAINNET,
+  RINKEBY,
+  ROPSTEN,
+} from '../../../../app/scripts/controllers/network/enums'
 
 const IncomingTransactionsController = proxyquire('../../../../app/scripts/controllers/incoming-transactions', {
   '../lib/random-id': { default: () => 54321 },
 }).default
 
-describe('IncomingTransactionsController', function () {
-  const EMPTY_INIT_STATE = {
+const FAKE_NETWORK = 'FAKE_NETWORK'
+const MOCK_SELECTED_ADDRESS = '0x0101'
+
+function getEmptyInitState () {
+  return {
     incomingTransactions: {},
     incomingTxLastFetchedBlocksByNetwork: {
-      [ROPSTEN]: null,
-      [RINKEBY]: null,
-      [KOVAN]: null,
       [GOERLI]: null,
+      [KOVAN]: null,
       [MAINNET]: null,
+      [RINKEBY]: null,
+      [ROPSTEN]: null,
     },
   }
+}
 
-  const NON_EMPTY_INIT_STATE = {
+function getNonEmptyInitState () {
+  return {
     incomingTransactions: {
       '0x123456': { id: 777 },
     },
     incomingTxLastFetchedBlocksByNetwork: {
-      [ROPSTEN]: 1,
-      [RINKEBY]: 2,
-      [KOVAN]: 3,
-      [GOERLI]: 5,
-      [MAINNET]: 4,
+      [GOERLI]: 1,
+      [KOVAN]: 2,
+      [MAINNET]: 3,
+      [RINKEBY]: 5,
+      [ROPSTEN]: 4,
```

Comment:
I noticed that the values for the networks differ from the mocks in `getNonEmptyInitStateWithFakeNetworkState`, if that matters.

Reply:
Ah, I don't think that matters. These are just arbitrary numbers to represent the last block fetched, so they don't need to be the same between mocks. I'll also be replacing `getNonEmptyInitStateWithFakeNetworkState` altogether in a subsequent PR (the one after the next one)

Reply:
Ah, I see. Never mind then! 👍